release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -65,6 +65,18 @@ This release adds improvements related to the following components and concepts:
 [id="ocp-release-notes-edge-computing_{context}"]
 === Edge computing
 
+[id="ocp-4-18-edge-computing-sno-shutdown_{context}"]
+==== Shutting down and restarting single-node clusters up to 2 years after cluster installation
```

Comment:
```suggestion
==== Shutting down and restarting single-node clusters up to 1 year after cluster installation
```
kubeconfig to approve CSRs is valid for 2 years, but we support SNOs shutdown for 1 year only

Reply:
@vrutkovs Okay thanks. I updated, but just to clarify, we are still talking about up til 1 year from the cluster installation, correct? (Not just a year in general of being shut down, no matter when they shut it down compared to the install date?)

Reply:
Yes, one year since cluster install